modules/plugin/imagemosaic/src/main/java/org/geotools/gce/imagemosaic/Utils.java

```diff
@@ -208,6 +212,51 @@
         MOSAIC_SUPPORT_FILES_FILTER = initMosaicSupportFilesFilter();
     }
 
+    /** Check if the provided reader is a MultiCRS Reader and it can support the specified crs. */
+    public static boolean isSupportedCRS(GridCoverage2DReader reader, CoordinateReferenceSystem crs)
```

Comment:
Do these methods need to be public?

Reply:
Yep, It will be used by WPS Raster Download